### PR TITLE
Improve guidance around semantic HTML

### PR DESCRIPTION
**Pull request by kjdchapman**
Many of these links aren't about semantic HTML, and I think a header explanation helps introduce this concept. 

I'm happy to take suggestions on resources, wording, etc.

### CODE REVIEW

File: core-skills/frontend-web-development/resources/semantic-html.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+
+Semantic HTML is the use of HTML markup emphasising the _meaning_ of the information that it contains.
+
+This is in contrast to markup that focuses on _presentation_ or _visual structure_.
+
+For example, a `<li>` (list) element shouldn't be used because you want bullet points - it should be used because you have items that _make sense_ in a list.
+
+For a second example, notice the introduction of `<strong>` and `<em>` tags, preferred over `<b>` and `<i>`, because they describe the intent - "I want to emphasis this content" rather than "I want this content to look a certain way".
+
+Semantic HTML has positive effects on accessibility, too - not everyone will consume your content the same way, so describing things more clearly will help.
```
Comment (SebAshton) [inline core-skills/frontend-web-development/resources/semantic-html.md]: If we're mentioning accessibility here, we should link to the accessibility article

File: core-skills/frontend-web-development/resources/semantic-html.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
-http://codeguide.co/#html
+# Resources on Semantic HTML
+
+Semantic HTML is the use of HTML markup emphasising the _meaning_ of the information that it contains.
+
+This is in contrast to markup that focuses on _presentation_ or _visual structure_.
+
+For example, a `<li>` (list) element shouldn't be used because you want bullet points - it should be used because you have items that _make sense_ in a list.
+
+For a second example, notice the introduction of `<strong>` and `<em>` tags, preferred over `<b>` and `<i>`, because they describe the intent - "I want to emphasis this content" rather than "I want this content to look a certain way".
```
Comment (SebAshton) [inline core-skills/frontend-web-development/resources/semantic-html.md]: Should mention the HTML5 block elements, and the semantic meaning they add, and why they are preferred to `div` soup or link to an article

File: core-skills/frontend-web-development/resources/semantic-html.md (hunk 1 above)
Comment (kjdchapman) [reply core-skills/frontend-web-development/resources/semantic-html.md]: Good idea!

Do you mean our page in the same guidance (https://learn.madetech.com/core-skills/frontend-web-development/resources/web-accessibility.html)? Or a specific article?

File: core-skills/frontend-web-development/resources/semantic-html.md (hunk 2 above)
Comment (kjdchapman) [reply core-skills/frontend-web-development/resources/semantic-html.md]: Good idea, I'll look for a good resource.

Comment (faissalMT) [inline core-skills/frontend-web-development/resources/semantic-html.md]: `<b>` and `<i>` [have semantic meanings in HTML5](https://softwareengineering.stackexchange.com/a/255588/309780)

Comment (kjdchapman) [reply core-skills/frontend-web-development/resources/semantic-html.md]: Huh, thanks, I didn't know!

I've actually ended up reworking this section, but it's good for me to know!

**Comment by kjdchapman** [comment]
Pushed another draft of this based on writing a presentation, this is now a bit... punchier.

**Pull request merged**